cheatsheets/Session_Management_Cheat_Sheet.md

```diff
@@ -246,7 +246,7 @@ Session IDs must be considered untrusted, as any other user input processed by t
 
 ### Renew the Session ID After Any Privilege Level Change
 
-The session ID must be renewed or regenerated by the web application after any privilege level change within the associated user session. The most common scenario where the session ID regeneration is mandatory is during the authentication process, as the privilege level of the user changes from the unauthenticated (or anonymous) state to the authenticated state. Other common scenarios must also be considered, such as password changes, permission changes or switching from a regular user role to an administrator role within the web application. For all these web application critical pages, previous session IDs have to be ignored, a new session ID must be assigned to every new request received for the critical resource, and the old or previous session ID must be destroyed.
+The session ID must be renewed or regenerated by the web application after any privilege level change within the associated user session. The most common scenario where the session ID regeneration is mandatory is during the authentication process, as the privilege level of the user changes from the unauthenticated (or anonymous) state to the unauthorized state. Other common scenarios must also be considered, such as password changes, permission changes or switching from a regular user role to an administrator role within the web application. For all these web application critical pages, previous session IDs have to be ignored, a new session ID must be assigned to every new request received for the critical resource, and the old or previous session ID must be destroyed.
```

Comment:
I do not like this change; it is harder to understand and it was not wrong, the state changed to authenticated.

Reply:
@mackowski 
Fair. however there is an argument that in context there is a distinction but generally speaking you are correct, they may seem the same.

That context is the heading `Renew the Session ID After Any Privilege Level Change` which essentially speaks to an Authorization mechanism. Therefore it is distinct from merely being authentic, to gain elevated access one must be authorised to that access level.

Reply:
Apologies that I did not respond for over a week on that...
You are right but it is still harder to understand by a regular IT person that does not work in security.
What about: state changed from the unauthenticated (or anonymous) to the authenticated, unauthorized state.

Reply:
No need to apologise, collaboration is like eventual consistency :+1: 
Very true, and good compromise. I feel silly for not realising it myself :facepalm: